toolchain/docs/idioms.md

```diff
@@ -266,17 +269,15 @@ This can be used for complex initialization, as in:
 ```cpp
 // variable declaration
 static const llvm::ArrayRef<std::byte> entropy_bytes =
-// initializer starts with a lambda
+    // initializer starts with a lambda
     []() -> llvm::ArrayRef<std::byte> {
-  static llvm::SmallVector<std::byte> bytes;
+      static llvm::SmallVector<std::byte> bytes;
 
-  // a bunch of code
-
-  // return the value to initialize the variable with
-  return bytes;
+      // a bunch of code
 
-// finish defining the lambda, and then immediately invoke it
-}();
+      // return the value to initialize the variable with
+      return bytes;
+    }();  // finish defining the lambda, and then immediately invoke it
```

Comment:
We try to avoid trailing line comments in the same way that Carbon disallows them completely.